.dev/compose.backbone.yml

```diff
@@ -56,18 +56,17 @@ services:
 
   seed-database:
     image: postgres
-    volumes:
-      - ./setup-postgres.sql:/app/setup-db/setup-postgres.sql
     environment:
       - PGPASSWORD=Passw0rd
-    command: psql -h postgres -U postgres -d enmeshed -f /app/setup-db/setup-postgres.sql
+      - CONSUMERAPI_VERSION=v4.2.6
```

Comment:
The consumer api version is duplicated now (line 3 and 61). Can we get rid of that duplication?

Reply:
we can, using compose vars.

Reply:
And do we want to do it? :)

Reply:
wo do 🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯